plugins/redis/secret_key.go

```diff
@@ -0,0 +1,55 @@
+package redis
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func SecretKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.SecretKey,
```

Comment:
Redis seems to call these "API Keys". Is there a specific reason we're not using `credname.APIKey` here?